import.py

```diff
@@ -962,7 +962,7 @@ def get_setting(key: str) -> Optional[str]:
                     "context": context,
                     "source_code": source,
                     "compiler": compiler_name,
-                    "compiler_flags": get_compiler_flags(compiler),
+                    "compiler_flags": settings["decompme"]["flags"] or get_compiler_flags(compiler),
```

Comment:
```suggestion
                    "compiler_flags": settings["decompme"].get("flags") or get_compiler_flags(compiler),
```
or this will crash if the property does not exist in the settings file. Or preferably, check `is not None` so that empty string does not get overridden.

Also, it would be good if you could use `json_dict()` and `assert isinstance(..., str)` so that mypy checks out.

Reply:
Thanks for the suggestions, I've made another attempt.